.github/workflows/deploy-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Existing tag to be released, e.g. v1.2.3, v2.2.2, v3.0.0'
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Deploy release
+    steps:
+      - name: Determine tag to be released
+        run: |
+          TAG=${GITHUB_REF:10}
```

Comment:
Why 10?

Reply:
Simple string indexing, so it strips the "refs/tags/" part from refs/tags/v5.4.1, needed to extract the tag

Reply:
thought so. can we add a small comment please so there is not future confusion?

Reply:
95b738f

Reply:
thanks.